backend/src/tasks/pools-updater.ts

```diff
@@ -98,7 +98,8 @@ class PoolsUpdater {
       logger.info(`Mining pools-v2.json (${githubSha}) import completed`, this.tag);
 
     } catch (e) {
-      this.lastRun = now - 600; // Try again in 10 minutes
+      // fast-forward lastRun to 10 minutes before the next scheduled update
+      this.lastRun = now - (config.MEMPOOL.POOLS_UPDATE_DELAY - 600);
```

Comment:
We might want to clamp `config.MEMPOOL.POOLS_UPDATE_DELAY - 600` to a minimum value to avoid continuous attempt without delay

EG: `Math.max(config.MEMPOOL.POOLS_UPDATE_DELAY - 600, 600)`